tests/fortran/test_fortran_client.py

```diff
@@ -91,3 +93,17 @@ def execute_cmd(cmd_list):
         print("OUTPUT:", output.decode("utf-8"))
         print("ERROR:", errs.decode("utf-8"))
         assert(False)
+
+@pytest.mark.skipif(
+    not test_gpu,
+    reason="SMARTREDIS_TEST_DEVICE does not specify 'gpu'"
```

Comment:
We should document this environment variable somewhere

Reply:
Turns out that most of the environment variables that control the behavior of testing are not in the documentation. I've added a brief description to the SmartRedis docs, but am deferring updates to the main documentation to another ticket.

Reply:
Ooh, good catch.  If you'll put in a placeholder for a section on environment variables, we can fill it in more fully when we clean up documentation in the release cycle.